bin/internal/update_dart_sdk.sh

```diff
@@ -56,20 +56,44 @@ if [ ! -f "$ENGINE_STAMP" ] || [ "$ENGINE_VERSION" != `cat "$ENGINE_STAMP"` ]; t
     echo
     exit 1
   }
-  >&2 echo "Downloading Dart SDK from Flutter engine $ENGINE_VERSION..."
 
-  # On x64 stdout is "uname -m: x86_64"
-  # On arm64 stdout is "uname -m: aarch64, arm64_v8a"
-  case "$(uname -m)" in
-    x86_64)
-      ARCH="x64"
-      ;;
-    *)
-      ARCH="arm64"
-      ;;
-  esac
+  OS="$(uname -s)"
```

Comment:
`uname -s` is already called up on line 28. Consider hoisting both calls up.

Reply:
good call, done.